devenv/Dockerfile

```diff
@@ -3,10 +3,10 @@ FROM ubuntu:latest
 ENV DEBIAN_FRONTEND=noninteractive
 
 WORKDIR /root/
-RUN apt-get update && apt-get install -y --no-install-recommends git apt-utils curl tar wget yamllint shellcheck flake8 golang python3-pip  && rm -rf /var/lib/apt/lists/*
-RUN wget -O ./install-required-packages.sh https://raw.githubusercontent.com/netdata/netdata/master/packaging/installer/install-required-packages.sh && chmod +x ./install-required-packages.sh && ./install-required-packages.sh netdata-all --non-interactive
+RUN apt-get update && apt-get upgrade -y && apt-get install -y --no-install-recommends git apt-utils curl tar wget yamllint shellcheck flake8 golang python3-pip ca-certificates  && rm -rf /var/lib/apt/lists/*
```

Comment:
We need a comment containing `hadolint ignore=DL3005,DL3008` here to suppress the (incorrect for our usage) warnings being listed for this line.